test cases/vala/11 mixed sources/meson.build

```diff
@@ -0,0 +1,5 @@
+project('foo', 'c', 'vala')
+
+glib = dependency('glib-2.0')
```

Comment:
actually I think once we specify 'vala' in langs, it should be automatically and passed over all vala sources